doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -81,16 +81,16 @@ For the avoidance of doubt, the following is considered to be **out of scope**:
 
-For example, whilst the management of VMs through Kubernetes is included, we will likely start with the "native" control of containers and add VMs at a later date - to be decided and documented in the Roadmap section below.
+For example, while the management of VMs through Kubernetes is included, the intention is to start with the "native" control of containers and add support for VMs at a later date.  The final decision will be determined and documented in the Roadmap section.
 
-In addition, we will start with a description of interfaces and capabilities (the "what") before at a later date providing guidance on "how" those elements are deployed (which will be documented in full detail in the Reference Implementation). In addition, an [Appendix](./appendix-a.md) will be created with the purpose of describing the transition from VNF to CNF and the potential pitfalls and complexities that may need consideration. This appendix may in turn lead to gaps that need filling in one or more Reference Architectures.
+This document will start with a description of interfaces and capabilities (the "what") before at a later date providing guidance on "how" those elements are deployed.  The details of how the elements will be used together will be documented in full detail in the Reference Implementation. In addition, an [Appendix](./appendix-a.md) is included with the purpose of describing the transition from VNF to CNF and the potential pitfalls and complexities that may need consideration. This appendix may in turn lead to the identification of gaps that need attention in one or more Reference Architectures.
 
 <a name="1.6"></a>
 ## 1.6 Roadmap
 
-As Kubernetes evolves over time it will add new technologies and capabilities and so the CNTT Reference Architecture will need to evolve with it. The final release cadence for the Reference Model changes and feature updates is still under discussion but the cadence of the Kubernetes Reference Architecture should at least follow the release cadence for Kubernetes, which is every three months and at least the support cadence, which is nine months (latest release, plus two previous releases).
+Kubernates is a relatively new technology (First release was in June 2014), so is still rapidly evolving. As Kubernetes changes over time it will be adding new capabilities and featues, so the CNTT Reference Architecture will need to evolve along with it. The final release cadence for the Reference Model changes and feature updates is still under discussion but the cadence of the Kubernetes Reference Architecture should at least follow the release cadence for Kubernetes, which is as of June 2020, every three months.  The back rev support again should match to the Kubernates cadence, which is nine months (latest release, plus two previous releases).
```

Comment:
I assume `back rev support` means "back revision support"?  Can this be clarified for those that aren't familiar with the term please?

Reply:
@tomkivlin fixed